doc/rose-rug-advanced-tutorials-suicide.html

```diff
@@ -260,6 +260,282 @@ <h3 class="alwayshidden">Note on suicide triggers (2)</h3>
         <li>triggering off a cleanup task in the suite</li>
       </ul>
     </div>
+
+    <div class="slide">
+          <h2 id="advanced-example">Advanced suicide triggers example</h2>
+      <p>The following example is one suite but with three possible 
+      scenarios:</p>
+
+      <dl>
+        <dd>If a member of a family fails carry on.
+        If a specified task fails go to the end of that cycle and remove the 
+        failed task from from the task pool.</dd>
+        <dd>Check the failure and if it meets the criteria then try a recovery 
+        task. 
+        If that task success then the suite carries on otherwise if it fails 
+        then the suite stops as it cannot be recovered.</dd>
+        <dd>If the first task fails go to the end of the cycle or if two 
```

Comment:
Might be more appropriate to lead with "If a task fails skip to the end of the cycle" as the general first line descriptor here.